authorization/v2/authorizer.go

```diff
@@ -105,7 +111,9 @@ func (a *authorizer) InvalidateServiceToken(ctx context.Context, token string) e
 	return a.tokenService.DeleteServiceToken(ctx, claims.Id)
 }
 
-func (a *authorizer) GetAuthorizedResources(token string, urisToCheck []common.UniformResourceIdentifier) ([]common.UniformResourceIdentifier, error) {
+func (a *authorizer) GetAuthorizedResources(ctx context.Context, token string, urisToCheck []common.UniformResourceIdentifier) ([]common.UniformResourceIdentifier, error) {
+	var err error
```

Comment:
nit: You probably don't need to declare the error here since you'd be declaring it in line 117 anyway, right?

Reply:
Good point, I'll fix